elements/lisk-api-client/src/transaction.ts

```diff
@@ -17,6 +17,7 @@ import {
 	signMultiSignatureTransaction,
 	computeMinFee,
 } from '@liskhq/lisk-transactions';
+import { Options } from '@liskhq/lisk-transactions/src/fee';
```

Comment:
Options should be exposed from the package. This will not work in the published package